logd/store/badger/history.go

```diff
@@ -164,6 +165,9 @@ func getEventWithCursors(event api.Event) (store.EventWithCursors, error) {
 	return eventWithCursors, nil
 }
 
+// TODO: RemoveOldEvents also needs to delete events over a certain age,
+// not just a count limit. This is necessary for cleanup of no-longer-tracked
+// log streams.
 const maxEventsPerStream = 5000
```

Comment:
I am considering indexing the logs on `(id, log)` instead of `(log, id)` since I think that it will be more common to try to scan (most or all) events in a given timeframe than to scan events from one specific log. That would allow the "delete older than" operation to be more efficient.
Either way, we'll need a secondary index, so I'll add that soon.

Reply:
👍🏻